contracts/other/notional/Join1155.sol

```diff
@@ -0,0 +1,122 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity 0.8.6;
+
+import "@yield-protocol/vault-interfaces/IJoin.sol";
+import "@yield-protocol/utils-v2/contracts/token/IERC20.sol";
+import "@yield-protocol/utils-v2/contracts/token/MinimalTransferHelper.sol";
+import "@yield-protocol/utils-v2/contracts/access/AccessControl.sol";
+import "@yield-protocol/utils-v2/contracts/math/WMul.sol";
+import "@yield-protocol/utils-v2/contracts/cast/CastU256U128.sol";
+import "./ERC1155.sol"; // TODO: Move to yield-utils-v2
+// ERC1155TokenReceiver is in ERC1155.sol
+
+contract Join1155 is IJoin, ERC1155TokenReceiver, AccessControl() {
+    using WMul for uint256;
+    using CastU256U128 for uint256;
+
+    event FlashFeeFactorSet(uint256 indexed fee);
+
+    bytes32 constant internal FLASH_LOAN_RETURN = keccak256("ERC3156FlashBorrower.onFlashLoan");
+    uint256 constant FLASH_LOANS_DISABLED = type(uint256).max;
```

Comment:
my lintere's complaining this doesn't have a visibility stated.  it's kinda weird to even have visibility on a constant as my understanding is that the value gets replaced during compile.  but we have visibility on the one above so at least I guess we should be consistent

Reply:
should this be a constant? What if we want to turn flashloans on?

Reply:
It's a valid concern. If no visibility is declared then `internal` is assumed. Once deployed, it would give some background as to why the flash loan fee is what it is.

Reply:
Fixed